docs/source/concepts/runner.rst

```diff
@@ -189,8 +189,45 @@ and used in the same service. For example:
 Custom Model Runner
 ^^^^^^^^^^^^^^^^^^^
 
-.. TODO::
-    Document creating custom Runnable with models from BentoML model store
+
+Custom Runnable built with Model from BentoML's model store:
+
+.. code::
+
+    from __future__ import annotations
+
+    from typing import Any
+
+    import bentoml
+    from bentoml.io import JSON
+    from bentoml.io import NumpyNdarray
+    from numpy.typing import ArrayLike
+
+
+    bento_model = bentoml.sklearn.get("spam_detection:latest")
+
+
+    class SpamDetectionRunnable(bentoml.Runnable):
+        SUPPORTS_CPU_MULTI_THREADING = True
```

Comment:
For future thought: should we have something like

```python
bentoml_models = (
    bentoml.sklearn.get("spam_detection:latest"),
)
```

that automatically passes the required models through to the runner?